include/deal.II/base/numbers.h

```diff
@@ -457,8 +455,8 @@ namespace numbers
      *
      * @note This function can also be used in CUDA device code.
      */
-    static constexpr DEAL_II_CUDA_HOST_DEV const number &
-                                                 conjugate(const number &x);
+    static constexpr KOKKOS_FUNCTION const number &
```

Comment:
Let's not use KOKKOS names and rather use use our own macros, which might be an alias to the KOKKOS one. There might be a time when we want to use something else maybe if some features have become part of the standard.

Reply:
What about renaming `DEAL_II_CUDA_HOST_DEV` to `DEAL_II_HOST_DEV` instead?